README.md

```diff
@@ -1,5 +1,38 @@
-# lighthouse 
-Controller to facilitate DNS discovery in multi-cluster connected environments (proof-of-concept state)
+# Lighthouse 
+The lighthouse along with Submariner facilitate DNS discovery in multi-cluster connected environments. The solution aims to compatible with any CNI-compatible cluster network provider.
+
+# Architecture
+The below digram shows the basic lighthouse architecture.
+
+![Lighthouse architecture](https://raw.githubusercontent.com/submariner-io/lighthouse/master/docs/img/architecture.png) 
+
+## Lighthouse Controller
+There is a central service discovery controller that gathers the information from the clusters, decides what information is to be shared and distributes the information as newly defined CRDs.
```

Comment:
"This is the central discovery controller..."

Reply:
Done.